src/message/action.js

```diff
@@ -879,8 +879,10 @@ function QueueReset() {
  * @property {String} Interface Interface
  * @augments Action
  */
-function QueueRemove() {
+function QueueRemove(queue, asteriskInterface) {
```

Comment:
Looking great! :) One minor thing: Shouldn't the order of the arguments be the interface first and the queue next? Same for QueueAdd. So it matches QueuePause and QueueUnpause